koku/api/report/queries.py

```diff
@@ -95,6 +95,16 @@ def check_view_filter_and_group_by_criteria(filter_set, group_by_set):
     return True
 
 
+def sanitize_tag(tag):
+    """Sanitize a tag by removing unwanted characters and URL-encoding it."""
+    characters_to_sanitize = r' "\'`;'
+    table = str.maketrans(characters_to_sanitize, "_" * len(characters_to_sanitize))
+    sanitized_tag = strip_prefix(tag, TAG_PREFIX).translate(table)
```

Comment:
Since `strip_prefix()` can be used to strip an arbitrary prefix, that should be done outside of this function. The caller should be responsible for calling that function separately from this function.
```suggestion
    sanitized_tag = tag.translate(table)
```